openff/toolkit/tests/test_molecule.py

```diff
@@ -302,28 +302,29 @@ def test_atom_constructor(self):
         atom1 = Atom(6, 1 * unit.elementary_charge, False)
         assert atom1.formal_charge == 1 * unit.elementary_charge
 
-    @pytest.mark.parametrize("atomic_number", range(1, 118))
+    @pytest.mark.parametrize("atomic_number", range(1, 117))
     def test_atom_properties(self, atomic_number):
         """Test that atom properties are correctly populated and gettable"""
         formal_charge = 0 * unit.elementary_charge
         is_aromatic = False
 
-        this_element = element(atomic_number)
+        expected_mass = MASSES[atomic_number]
+        expected_symbol = SYMBOLS[atomic_number]
 
         atom = Atom(
-            this_element.atomic_number,
+            atomic_number,
             formal_charge,
             is_aromatic,
-            name=this_element.name,
+            name="fOO",
         )
-        assert atom.atomic_number == this_element.atomic_number
-        # Equality comparison fails with mendeleev v0.9.0 and older,
-        # should be fixed when a new release is made
+        assert atom.atomic_number == atomic_number
+        # this API point might be removed?
         # assert atom.element == this_element
         assert atom.formal_charge == formal_charge
         assert atom.is_aromatic == is_aromatic
-        assert atom.name == this_element.name
-        assert atom.mass == this_element.mass
+        assert atom.symbol == expected_symbol
+        assert atom.mass == expected_mass
```

Comment:
Since our use of `atom.mass` below requires it to always be in Daltons/amus, let's add a test for that here.

Reply:
This might be implicitly covered, but I've added an explicit check